docs/en/query_language/functions/ip_address_functions.md

```diff
@@ -1,18 +1,18 @@
-# Functions for working with IP addresses
+# IP函数
```

Comment:
A wrong file?
`docs/en/query_language/functions/ip_address_functions.md` -> `docs/zh/query_language/functions/ip_address_functions.md`

Reply:
Thanks, it has been fixed.